api/src/test/java/com/nicknathanjustin/streamercontracts/votes/VoteServiceImplTest.java

```diff
@@ -49,71 +50,97 @@ public void recordVote_nullContractModel_throwsException() {
 
     @Test(expected = NullPointerException.class)
     public void isVotingComplete_nullContractModel_throwsException() {
-        voteServiceImpl.isVotingComplete(null);
+        Optional<VoteModel> optionalProposerVote = Optional.empty();
+        Optional<VoteModel> optionalStreamerVote = Optional.empty();
```

Comment:
nit: final on both

Reply:
Would be nice to have tests that check for NPE on null optionalProposerVote  and optionalStreamerVote  inputs. Its a small quick test and can keep someone from breaking something:

```
isVotingComplete_nullOptionalProposerVote _throwsException
```